src/waveform/renderers/allshader/waveformrendererfiltered.cpp

```diff
@@ -188,9 +188,9 @@ bool WaveformRendererFiltered::preprocessInner() {
         xVisualFrame += visualIncrementPerPixel;
     }
 
-    DEBUG_ASSERT(reserved ==
-            vertexUpdater[0].index() + vertexUpdater[1].index() +
-                    vertexUpdater[2].index());
+    //     DEBUG_ASSERT(reserved ==
+    //             vertexUpdater[0].index() + vertexUpdater[1].index() +
+    //                     vertexUpdater[2].index());
```

Comment:
anything to do here? Is this fixed by / in conflict with #14470?

Reply:
Ah yeah, this is fixed by that one